pkg/inspector/image-inspector_test.go

```diff
@@ -143,7 +151,7 @@ func TestGetAuthConfigs(t *testing.T) {
 	}
 
 	for k, v := range tests {
-		ii := &defaultImageInspector{*v.opts, InspectorMetadata{}}
+		ii := &defaultImageInspector{*v.opts, iiapi.InspectorMetadata{}, &MockImageServer{}}
```

Comment:
The tests pass with `nil`, in [1] we check to see if the image server is defined. I will remove this redundant code for the mock server. It is easy to make again but I don't think it will be needed here anyway.

[1]https://github.com/openshift/image-inspector/pull/39/files#diff-cb14c05f76ccda787aced81d2a624e69R138